source/features/keyboard-navigation.tsx

```diff
@@ -38,18 +21,17 @@ function runShortcuts(event: KeyboardEvent): void {
 	if (['j', 'k'].includes(event.key)) {
 		event.preventDefault();
 
-		const items = select.all('.js-targetable-element')
-			.filter(element => {
-				if (element.id.startsWith('pullrequestreview-')) {
-					return pullRequestReviewHasComment(element);
-				}
-
-				if (element.classList.contains('js-minimizable-comment-group')) {
-					return !isCommentGroupMinimized(element);
-				}
-
-				return true;
-			});
+		const items = select
+			.all([
+				// Files in diffs
+				'.js-targetable-element[id^="diff-"]',
+				// Comments (to be `.filter()`ed)
+				'.js-minimizable-comment-group'
+			])
+			.filter(element =>
+				!element.classList.contains('js-minimizable-comment-group') ||
```

Comment:
It can't select `.js-minimizable-comment-group` and then exclude it, it must be wrong.

Also, if necessary, such simple check should just be part of the selector: `.original-selector:not(.something-to-exclude)`